package.json

```diff
@@ -128,13 +128,6 @@
     "wait-on": "^3.2.0",
     "workerize-loader": "^1.1.0"
   },
-  "husky": {
-    "hooks": {
-      "commit-msg": "node scripts/commit-validator.js",
-      "pre-commit": "lint-staged",
-      "pre-push": "yarn jest"
-    }
-  },
```

Comment:
Looks like you were struggling with our precommit validation.  Likely the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard that's being enforced.  It's a bit inconvenient, but enables some automation features.  

In your case, you should be able to commit with a message like "feat(Panel): passing portalMount to Panel" or "chore(Paprika): restore husky config" 😉 

And if you're really stuck by this, you can commit with `--no-verify` to skip the hooks.